pkg/install/v1alpha1/jiva_volume.go

```diff
@@ -672,7 +669,7 @@ metadata:
 spec:
   meta: |
     id: createpatchrep
-    runNamespace: {{ .Volume.runNamespace }}
+    runNamespace: {{ .Config.JivaRunNamespace.value }}
```

Comment:
we need to be patching the labels of nodes as well

Reply:
I'm not getting you here, why do we need to patch the labels of nodes ?